### PR TITLE
hv:replace unsigned long long with uint64_t

### PR DESCRIPTION
**Pull request by mingqiangchi**
unsigned long long--> uint64_t
long long --> int64_t

Signed-off-by: Mingqiang Chi <mingqiang.chi@intel.com>

### CODE REVIEW

**Comment by NanlinXie** [comment]
Can one of the admins verify this patch?

File: hypervisor/lib/sprintf.c [hunk 1]
```diff
@@ -515,19 +515,19 @@ int do_print(const char *fmt, struct print_param *param,
 						(param->vars.flags &
 						PRINT_FLAG_LONG_LONG) ?
 						__builtin_va_arg(args,
```
Comment (dongyaozu) [inline hypervisor/lib/sprintf.c]: For this one, it relies on int/long/long long... I am not sure if we should change.. Can you check with Jason? Same for others in __builtin_va_arg.

**Comment by lijinxia** [comment]
OK to verify

**Comment by mingqiangchi** [comment]
have checked with Jason, will not change  __builtin_va_arg

**Comment by lijinxia** [comment]
Verified+1

**Pull request merged**